package.json

```diff
@@ -65,6 +68,7 @@
     "brfs": "1.4.3",
     "csscolorparser": "1.0.3",
     "earcut": "2.1.1",
+    "exorcist": "^0.4.0",
```

Comment:
This line is duped below. I also prefer to peg the version numbers to exact ones (instead of using `~` or `^`). It's more of a pain to upgrade dependencies, but we've been bitten multiple times by auto-upgrades of packages that were supposed to be backwards compatible but included regressions -- so we switched to exact versions.